client.go

```diff
@@ -414,7 +415,14 @@ func (c *Client) Deploy(path string) (err error) {
 	}
 
 	// Push the image for the named service to the configured registry
-	if err = c.pusher.Push(f); err != nil {
+	imageDigest, err := c.pusher.Push(f)
+	if err != nil {
+		return
+	}
+
+	// Store the produced image Digest in the config
+	f.ImageDigest = imageDigest
+	if err = writeConfig(f); err != nil {
```

Comment:
It is not needed to store the digest curretnly (the push is immediately followed by deploy), so I can remove this part. But I thought we might want to use that in the future, if we introduce a ways how to fire `push` `deploy-only` etc (as was discussed on Slack).